package.json

```diff
@@ -145,6 +146,7 @@
     "electron-fetch": "^1.7.4",
     "electron-log": "^4.4.8",
     "electron-squirrel-startup": "^1.0.0",
+    "electron-store": "^8.1.0",
```

Comment:
Should we store data in postgres? We will need to hold a registry of tables sooner or later.

Reply:
in light of the other comment - I think I'd rather avoid state date in PG and prefer this. In case we do end up storing some complicated state in PG, then we might consider dropping `electron-store` for PG.

As much as I don't enjoy a new dep, putting just a single trivial flag in PG feels overkill.